src/recovery/impl/thread_local_logger.cpp

```diff
@@ -171,6 +184,16 @@ EpochNumber ThreadLocalLogger::GetMinDurableEpochForAllThreads() {
   return min_flushed_epoch;
 }
 
+std::string ThreadLocalLogger::GetLogFileName(size_t thread_id) {
+  // TODO: think of beautiful path concatation in C++
```

Comment:
We can use `std::filesystem::path` instead of `std::string`.
ref: https://en.cppreference.com/w/cpp/filesystem/path

It would be nice to convert all `std::string` as filepaths to `std::filesystem::path`; but you don't have to do that in this PR; we'll add a future issue to do this.